src/map.js

```diff
@@ -357,14 +341,18 @@ var map = function (arg) {
   /**
    * Get/Set zoom level of the map
    *
-   * @param {number} val if undefined, return the current zoom level.
+   * @param {number} [val] if undefined, return the current zoom level.
```

Comment:
Let us do that, too.  I'd say the only exception is when the first word is a literal, in which case it can be in monospace.